### PR TITLE
fix: pluginMinifyHtmlLiterals

### PR DESCRIPTION
**Pull request by vegarringdal**
fixed the pluginMinifyHtmlLiterals

### CODE REVIEW

File: src/plugins/core/plugin_minify_html_literals.ts [hunk 1]
```diff
@@ -10,43 +10,43 @@ interface IPluginMinifyHtmlLiterals {}
...
+      // verify if module is installed
+      if (!isNodeModuleInstalled('minify-html-literals')) {
+        ctx.fatal(`Fatal error when capturing ${module.absPath}`, [
+          'Module "minify-html-literals" is required, Please install it using the following command',
+          'npm install minify-html-literals --save-dev',
+        ]);
+        return;
+      }
+      // check if we have any html`` or css`` template tags
+      if (/\html`|css`/.test(module.contents)) {
```
Comment (nchanged) [inline src/plugins/core/plugin_minify_html_literals.ts]: gotta check for `module.isExecutable` here

Comment (vegarringdal) [reply src/plugins/core/plugin_minify_html_literals.ts]: K, ill have a look tomorrow after work

Comment (vegarringdal) [reply src/plugins/core/plugin_minify_html_literals.ts]: fixed... wil be very late if I wait until tomorrow :-)
time for bed :-)

**Comment by vegarringdal** [comment]
@nchanged 
All ok now?
Tried it on my project and its minifying

**Comment by nchanged** [comment]
@vegarringdal I am actually thinking it's much better to do it with a transformer. But I will merge it now, and after the docs have been completed we will re-visit this one ;-)

**Pull request merged**